lib/resources/datav2/rest_v2.js

```diff
@@ -77,6 +100,37 @@ async function* getTrades(symbol, options, config) {
   }
 }
 
+async function getMultiTrades(symbols, options, config) {
+  if (!Array.isArray(symbols)) {
+    throw new Error("symbols should be an array");
+  }
+  Object.assign(options, { symbols: symbols.join() });
+  const multiTrades = getMultiDataV2(TYPE.TRADES, options, config);
+  let trades = {};
+  for await (let t of multiTrades) {
+    if (!trades[t.symbol]) {
+      trades[t.symbol] = [entityv2.AlpacaTradeV2(t.data)];
+    } else {
+      tradesSoFar = trades[t.symbol];
+      tradesSoFar.push(entityv2.AlpacaTradeV2(t.data));
+      trades[t.symbol] = tradesSoFar;
```

Comment:
Is this really the best way to do this in javascript?

Reply:
could use the spread operator
```js
trades[t.symbol] = [...trades[t.symbol], entityv2.AlpacaTradeV2(t.data)]
```

Reply:
thank you for your suggestion!